docs/source/tutorial.rst

```diff
@@ -29,10 +29,17 @@ To run this tutorial, you'll need to:
 
        - Install MLflow and scikit-learn. There are two options for installing these dependencies:
 
-           1. Install MLflow with extra dependencies, including scikit-learn
-              (via ``pip install mlflow[extras]``)
-           2. Install MLflow (via ``pip install mlflow``) and install scikit-learn separately
-              (via ``pip install sckit-learn``)
+         - Install MLflow with extra dependencies (``pip install mlflow[extras]``), where the extra dependencies are:
```

Comment:
Can we link the user to the set of dependencies contained in the quickstart's "Installing MLflow" section instead of listing all of the dependencies again? My concern is that these dependencies may change over time and we might forget to update the list in multiple places.

Reply:
Done.